### PR TITLE
updates set

### PR DESCRIPTION
**Pull request by m4sterbunny**
- Updates Readme to remove POC data
- Updates set to quiet false positives
- Adds links back to docs-template for VS code integration

### CODE REVIEW

File: spelling/README.md [hunk 1]
```diff
@@ -48,15 +48,13 @@ Vale is highly customizable and the boilerplate styles may not be what you need.
...
 
-The [vale.ini](vale.ini) file provides various switches to turn styles on and off and to set what file types are formatted. Furthermore, as part of the GHA, the downstream repos that use this can specify which folders Vale may lint.
-
-## To do
+The [vale.ini](.vale.ini) file provides various switches to turn styles on and off and to set what file types are formatted. Furthermore, as part of the GHA, the downstream repos that use this can specify which folders Vale may lint.
 
 ## Vale and YAML
 
-Vale might need a parser to handle linting YAMLs (e.g., PyYAML, ruyaml). Then update the `.vale.ini` file to include the YAML style guide and specify the file extensions to lint.
+Vale needs a parser to handle linting YAMLs. This is (possibly/probably) why there is an additional ini file [vale-star.ini](vale-star.ini).
```
Comment (m4sterbunny) [inline spelling/README.md:55]: @joshuafernandes no longer convinced this is so -- looking at your vale-star.ini, did you actually apply parser too or did you just manage to tell Vale to look a YAML files?! If the latter, there is no need to duplicate the file is there --> as in, we should just be able to add file types to lint as list or array??

**Review by joaniefromtheblock — APPROVED** [review]
lgtm

File: spelling/README.md (hunk 1 above)
Comment (joshuafernandes) [reply spelling/README.md:59]: A bit of both. Vale is built for docs with some sort of markup (md, html, xml, rst etc) and json, yaml are data formats not really markup. In the strict sense vale doesn't parse yml or json, however there is a fall through case where it handles files (anything) as text - this is what we use with the vale-star.ini file

No special scoping is taken into account though, and yml doesn't cater to scoping anyway so this works well.

File: spelling/README.md [hunk 2]
```diff
@@ -48,56 +50,10 @@ Vale is highly customizable and the boilerplate styles may not be what you need.
...
-Goerli
-on-chain
-
-Shouldn't trigger
-
-booleans
-celo
-
-
+Vale needs a parser to handle linting YAMLs. This is (possibly/probably) why there is an additional ini file [vale-star.ini](vale-star.ini).
```
Comment (m4sterbunny) [inline spelling/README.md:59]:
```suggestion
In the strict sense vale doesn't parse yaml or json, however, there is a fall through case where it handles files (anything) as text. This is used with the [vale-star.ini](vale-star.ini).
```

**Review by m4sterbunny — COMMENTED** [review]
Can I do a self review?

**Review by alexandratran — APPROVED** [review]

**Pull request merged**